cms/jinja2/templates/components/streamfield/stream_block--related-links.html

```diff
@@ -1,4 +1,4 @@
-<ol class="ons-list ons-list--bare ons-u-bt ons-u-mb-xl">
```

Comment:
This is different to how the related links are presented on the analysis page. They are using slightly different streamfield blocks. I'm unsure if there was a deliberate reason for this so have left them for now, but they may need rationalising.